app/uk/gov/hmrc/agentmappingfrontend/config/frontendAppConfig.scala

```diff
@@ -10,15 +29,34 @@ trait AppConfig {
   val reportAProblemNonJSUrl: String
 }
 
-object FrontendAppConfig extends AppConfig with ServicesConfig {
+trait StrictConfig{
+  def loadConfig(key: String): String = configuration.getString(key).getOrElse(throw new Exception(s"Missing configuration key: $key"))
+}
 
-  private def loadConfig(key: String) = configuration.getString(key).getOrElse(throw new Exception(s"Missing configuration key: $key"))
+object GGConfig extends StrictConfig {
```

Comment:
Consistency is good but in this case the config id needed by the AuthActions trait, and the price of making its injectable is additional parameters elsewhere.